lib/gollum-lib/markups.rb

```diff
@@ -59,5 +65,8 @@ class Markup
              :extensions => ['mediawiki','wiki'], :reverse_links => true)
     register(:pod, "Pod",
              :enabled => MarkupRegisterUtils::executable_exists?("perl"))
+    register(:bib, "BibTeX", :extensions => ['bib'],
+             :enabled => MarkupRegisterUtils::gems_exist?(["bibtex-ruby", "citeproc-ruby", "csl-styles"]),
```

Comment:
I really like the idea of having an info page, but the worry of course is that all users would be able to view it. Not sure how I feel about that.